### PR TITLE
309294-Configure-embed-configjson-instead-of-embed-properties-in-blazor-with-js

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
+  * Please ensure that you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.
+
+    ![Embed Settings](https://github.com/boldbi/aspnet-core-sample/assets/91586758/b3a81978-9eb4-42b2-92bb-d1e2735ab007)
+
+  * To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+  
+    ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
+    ![EmbedConfig Properties](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d6ce925a-0d4c-45d2-817e-24d6d59e0d63)
+
+  * Copy the downloaded embedConfig.json file and paste it into the designated [location](https://github.com/boldbi/blazor-server-sample) within the application. Please ensure that you have placed it in the application as shown in the following image.
```
Comment (vinothkumar1732) [inline README.md]: Highlight the **embedConfgi.json** as in previous step.

File: Program.cs [hunk 2]
```diff
@@ -1,28 +1,39 @@
-using System;
-using System.Collections.Generic;
+using BoldBIEmbedSample.Data;
```
Comment (vinothkumar1732) [inline Program.cs:1]: Remove the unwanted files from project like csproj.user file, bin, obj and package-lock.json.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Highlighted the embedConfgi.json point

File: Program.cs (hunk 2 above)
Comment (ghost) [reply Program.cs:1]: Unwanted files are removed.

File: README.md [hunk 3]
```diff
@@ -1,63 +1,63 @@
...
- ## Using the Blazor server sample
+ * https://help.boldbi.com/embedded-bi/faq/where-can-i-find-the-product-version/
+
+ #### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox, and Safari.
+
+ ## Configuration
+
+  * Please ensure that you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.
```
Comment (florenceliliana) [inline README.md]: * Please ensure you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.

File: README.md [hunk 4]
```diff
@@ -1,63 +1,63 @@
...
+
+    ![Embed Settings](https://github.com/boldbi/aspnet-core-sample/assets/91586758/b3a81978-9eb4-42b2-92bb-d1e2735ab007)
+
+  * To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+  
+    ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
+    ![EmbedConfig Properties](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d6ce925a-0d4c-45d2-817e-24d6d59e0d63)
+
+  * Copy the downloaded `embedConfig.json` file and paste it into the designated [location](https://github.com/boldbi/blazor-server-sample) within the application. Please ensure that you have placed it in the application as shown in the following image.
```
Comment (florenceliliana) [inline README.md]: * Copy the downloaded `embedConfig.json` file and paste it into the designated [location](https://github.com/boldbi/blazor-server-sample) within the application. Please ensure you have placed it in the application, as shown in the following image.

File: README.md [hunk 5]
```diff
@@ -1,63 +1,63 @@
...
+  2. To run the application, use the command `dotnet watch run` in the terminal. After executing the command, the application will automatically launch in the default browser. You can access it at the specified port number (e.g., https://localhost:5001).
 
+     ![dashboard image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
 
 Please refer to the [help documentation](https://help.boldbi.com/embedded-bi/javascript-based/samples/v3.3.40-or-later/blazor-with-javascript/#how-to-run-blazor-server-sample) to know how to run the sample.
 
-## Online Demos
+## Important notes
+
+It is recommended to not store passwords and sensitive information in configuration files for security reasons, in a real-world application. Instead, you should consider using a secure application, such as Key Vault, to safeguard your credentials.
```
Comment (florenceliliana) [inline README.md]: It is recommended not to store passwords and sensitive information in configuration files for security reasons in a real-world application. Instead, you should consider using a secure application, such as Key Vault, to safeguard your credentials.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: Content updated

File: README.md (hunk 4 above)
Comment (ghost) [reply README.md]: Content updated

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md]: Content updated

**Review by florenceliliana — APPROVED** [review]

**Review by Alagarsamyd — APPROVED** [review]

**Pull request merged**